tests/container_checker/test_container_checker.py

```diff
@@ -156,19 +156,18 @@ def postcheck_critical_processes_status(duthost, up_bgp_neighbors):
                       post_test_check, duthost, up_bgp_neighbors)
 
 
-def stop_container(duthost, container_name):
+def stop_container(duthost, container_name, service_name):
     """Stops the running container.
 
     Args:
       duthost: Host DUT.
-      container_name: A string represents the container which will be stopped.
+      service_name: A string represents the container which will be stopped.
 
     Returns:
       None
     """
-
     logger.info("Stopping the container '{}' on DuT '{}' ...".format(container_name, duthost.hostname))
-    duthost.shell("sudo systemctl stop {}.service".format(container_name))
+    duthost.shell("sudo systemctl stop {}.service".format(service_name))
```

Comment:
instead of adding new logic can we see if we use API `stop_service` ?  It takes care of asic internally

Reply:
Modified to use stop_service. 
Because the container names are selected using enum_dut_feature_container fixture, we have to still find the asic_index from container name. So kept that logic to find the asic_index and then invoke stop_service.